docs/DeveloperGuide.md

```diff
@@ -73,30 +73,31 @@ Note: The C symbols are a result of the PlantUml layout.
 ### Pairing Component
 API: [`pairingList.java`](../src/main/java/seedu/duke/PairingList.java)
 
-* `PairingList` is responsible for recording which clients renting which property.
+* `PairingList` is responsible for recording which clients are renting which property.
```

Comment:
Do consider the following: `PairingList` is responsible for recording clients and their respective rented properties.